tests/test_transform.py

```diff
@@ -0,0 +1,90 @@
+"""
+Test script for extract_production.py
+"""
+import datetime
+import pandas as pd
+from pipeline.transform import Transform
+
+transform = Transform()
+
+
+class TestTransform:
```

Comment:
This is really thorough so let's just add a doc string to the class, the functions themselves are fairly easy to understand.